packages/forma-36-website/src/components/Docsearch.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import { css } from '@emotion/core';
+import tokens from '@contentful/forma-36-tokens';
+import { TextInput, Icon } from '@contentful/forma-36-react-components';
+
+const styles = {
+  container: css`
+    position: relative;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 0.5rem;
```

Comment:
Please use a F36 token here

```suggestion
    margin-bottom: ${tokens.spacingXs};
```